test/sql/parallelism/interquery/concurrent_attach_detach.cpp

```diff
@@ -246,11 +270,10 @@ TEST_CASE("Run a concurrent ATTACH/DETACH scenario", "[attach][.]") {
 
 	execQuery(init_conn, "SET catalog_error_max_schemas = '0'");
 	execQuery(init_conn, "SET threads = '1'");
-	// execQuery(initConn, "SET default_block_size = '16384'");
-	// execQuery(initConn, "SET storage_compatibility_version = 'v1.3.2'");
+	execQuery(init_conn, "SET storage_compatibility_version = 'v1.3.2'");
```

Comment:
Can we move the test group to `"[interquery][.]"` so this gets run in the thread sanitizer?